Makefile

```diff
@@ -35,12 +35,13 @@ update-forest:
 	git submodule update --init --recursive
 
 modify-forest:
-	sed -i -e 's|fil_actor_interface = "2"|fil_actor_interface = { path = "../fil_actor_interface" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actors_shared = "2"|fil_actors_shared = { path =  "../fil_actors_shared" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actor_account_state = "2"|fil_actor_account_state = { path =  "../actors/account" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actor_init_state = "2"|fil_actor_init_state = { path =  "../actors/init" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actor_miner_state = "2"|fil_actor_miner_state = { path =  "../actors/miner" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actor_power_state = "2"|fil_actor_power_state = { path =  "../actors/power" }|g' ./forest/Cargo.toml
-	sed -i -e 's|fil_actor_system_state = "2"|fil_actor_system_state = { path =  "../actors/system" }|g' ./forest/Cargo.toml
+	sed -i -e 's|fil_actor_interface = "4"|fil_actor_interface = { path = "../fil_actor_interface" }|g' ./forest/Cargo.toml
```

Comment:
Do we have to specify the version here? Can't we match any version?

Reply:
fixed